sql.txt

```diff
@@ -1,14 +1,23 @@
 (1) Using the Customer Table, select the CustomerName, ContactName, and Country.  Use ORDER BY to order by Country.  
 Use LIMIT and OFFSET to get entries 11 through 20. Paste your SQL statement below. 
+    select CustomerName, ContactName, Country from Customers LIMIT 10 OFFSET 10;
 
 (2) Select all columns from the Customer table where the ContactName starts with A.  Paste your SQL statement below.
+    select * from Customers where ContactName like "A%";
 
 (3) Select all columns from the OrderDetails table where the ProductID is 51 and the quantity is greater than 10.
 Paste your SQL statement below.
+    select * from OrderDetails where ProductID = 51 and quantity > 10;
 
 (4) Insert 3 rows into the Products table.  Note that you will have to specify a valid SupplierID and CategoryID, 
 corresponding to rows from the Supplier and Category tables.  Paste your three SQL statements below.
+    Insert into Products(ProductID, ProductName, SupplierID, CategoryID, Unit, Price) VALUES(78, 'Flax Energy Bars', 25, 5, '5 boxes x 10 bags', 30) ;
+    Insert into Products(ProductID, ProductName, SupplierID, CategoryID, Unit, Price) values(79, 'Madras Mixture', 20, 5, '10 - 3oz boxes', 20);
+    insert into products(ProductID, ProductName, SupplierID, CategoryID, Unit, Price) values(80, 'Motichur Laddu', 20, 3, '10 boxes', 50 );
```

Comment:
These commands work individually. But I would suggest using the simpler syntax for example: 
`INSERT INTO table_name (column_list)
VALUES
    (value_list_1),
    (value_list_2),
    (value_list_3);` 
  Good job! Keep it up.

Reply:
Thank you. Got it. I will make a note for myself.